routemaster/state_machine/api.py

```diff
@@ -88,6 +88,7 @@ def create_label(app: App, label: LabelRef, metadata: Metadata) -> Metadata:
                 name=label.name,
                 state_machine=state_machine.name,
                 metadata=metadata,
+                updated=func.now(),
```

Comment:
This seems easy to miss when adding a new edit. Can we mitigate that by automating it?

One option would be to use a database trigger, though I'm not sure whether SqlAlchemy supports them?

Reply:
I think SQLAlchemy supports them, if only through raw SQL, but there aren't many places we update labels, so for now I think it's best to just do it manually.